dists/tvos/Info.plist.in

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
Since this is a template file, please add it to `devtools/update-version.pl` to the ` @subs_files` list.